Tests/SonarScanner.MSBuild.PreProcessor.Tests/DataModel/QualityProfile.cs

```diff
@@ -49,23 +49,23 @@ public QualityProfile AddProject(string projectKey, string projectBranch = null)
             return this;
         }
 
-        public QualityProfile AddRule(ActiveRule rule)
+        public QualityProfile AddRule(SonarRule rule)
         {
             ActiveRules.Add(rule);
             return this;
         }
 
-        public QualityProfile AddInactiveRule(string ruleKey)
+        public QualityProfile AddInactiveRule(SonarRule rule)
```

Comment:
Are these two methods actually used? I see in the next file we are directly adding items directly to the collections.